plugins/cuckoo/plugin.spec.yaml

```diff
@@ -5,7 +5,8 @@ name: cuckoo
 title: Cuckoo Sandbox
 description: "Cuckoo Sandbox is an open source automated malware analysis system. Using the Cuckoo Sandbox plugin for
 Rapid7 InsightConnect, users can analyze files and URLs, manage tasks, and more"
-version: 1.0.2
+version: 2.0.0
```

Comment:
Can this be 1.0.3 since we are not breaking backwards compatibility?

Reply:
The validators thought it should be a major as I changed the title of one of the inputs from "Error message" to "Error Message"